migrator/src/test/java/com/google/phonenumbers/migrator/MetadataReaderTest.java

```diff
@@ -0,0 +1,14 @@
+package com.google.phonenumbers.migrator;
```

Comment:
You can create a couple of test metadata files if you want that you can use to test these. Under this testing dir, you can create a dir called 'testdata' and create a couple metadata files. Maybe one invalid one as well And test taht you are getting what you expect.